### PR TITLE
[WM-1468] Leo PR action with no inputs

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

**Review by cjllanwarne — APPROVED** [review]
Looks awesome!

File: .github/workflows/make_leo_pr.yml [hunk 1]
```diff
@@ -29,14 +36,36 @@ jobs:
         env:
           BROADBOT_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
         run: |
+          function getVersionFromReference(){
+            cat $1 | \
+            sed -e '/cromwellApp/,/chartVersion/!d' -e 's:^[ ]*::' | \
```
Comment (cjllanwarne) [inline .github/workflows/make_leo_pr.yml:41]: Maybe a comment explaining what this is doing, and what type of file to apply it to?

File: .github/workflows/make_leo_pr.yml [hunk 2]
```diff
@@ -29,14 +36,36 @@ jobs:
...
+            tail -1 | \
+            awk -F'"' '{print $2}'
+          }
+
+          function getVersionFromDocker(){
+            grep "ENV CROMWELL_CHART_VERSION" $1 | awk -F'[ ]' '{ print $3 }'
```
Comment (cjllanwarne) [inline .github/workflows/make_leo_pr.yml:47]: Maybe a comment explaining what this is doing, and what type of file to apply it to?

File: .github/workflows/make_leo_pr.yml [hunk 3]
```diff
@@ -57,7 +93,7 @@ jobs:
           github-token: ${{ secrets.BROADBOT_TOKEN }}
           script: |
             const result = await github.rest.pulls.create({
-              title: '${{ github.event.inputs.jira_ticket }}: Update Cromwell version to ${{ github.event.inputs.new_cromwell_helm_version }}',
+              title: '[AUTOMERGE] Update Cromwell version to ${NEW_CROMWELL_HELM_V}',
```
Comment (salonishah11) [inline .github/workflows/make_leo_pr.yml:96]: Do we need to have ticket number associated with PRs that are auto-merged?

**Review by salonishah11 — APPROVED** [review]

File: .github/workflows/make_leo_pr.yml (hunk 3 above)
Comment (mspector) [reply .github/workflows/make_leo_pr.yml:96]: great question - Chris has observed that similar tickets into terra-helmfile didn't seem to have them in their titles, so I think we're ok? 

(there's a link in the body of the ticket: https://broadworkbench.atlassian.net/browse/WM-1468)

Comment (salonishah11) [reply .github/workflows/make_leo_pr.yml:96]: Sounds good 👍

**Pull request merged**